### DISCUSSION TITLE
Does anything new coming in AdonisJS 5?

### DISCUSSION THREAD

**Discussion by hiteshmungpara** [adonisjs/core#4038] [General]
Any new features coming in current version?

**Comment by Barbapapazes** [comment]
Adonis6 is on the way!

**Reply by McSneaky** [reply]
Yeah, 6 is on the way that does a lot of internal cleanup + makes package creation and maintaining a lot easier

When v6 has landed, then we can also start seeing some more features and quite some improvements :) 

v6 doesn't have much breaking changes tho, so it's easy to move v5 project over to v6
